tools/uniswap/build-uniswap-list.ts

```diff
@@ -0,0 +1,122 @@
+import fs from "fs";
+import axios from "axios";
+
+// USAGE: ts-node tools/uniswap/build-uniswap-list.ts <PATH_TO_JSON_WITH_ALL_PAIRS>
+
+const AAVE_TOKEN_LIST = "https://tokenlist.aave.eth.link/";
+const UNISAP_PAIRS = "https://raw.githubusercontent.com/jab416171/uniswap-pairtokens/master/uniswap_pair_tokens.json";
```

Comment:
Can we keep it locally, I don't want to be dependent on untrusted 3rd party repo.